xmodule/tests/test_course_module.py

```diff
@@ -32,10 +33,13 @@
 _NEXT_WEEK = _TODAY + timedelta(days=7)
 
 
-class CourseFieldsTestCase(unittest.TestCase):
+class CourseFieldsTestCase(unittest.TestCase):  # lint-amnesty, pylint: disable=missing-class-docstring
 
     def test_default_start_date(self):
-        assert xmodule.course_module.CourseFields.start.default == datetime(2030, 1, 1, tzinfo=utc)
+        assert xmodule.course_module.CourseFields.start.default == DEFAULT_START_DATE
+
+    def test_default_enrollment_start_date(self):
+        assert xmodule.course_module.CourseFields.start.default == DEFAULT_START_DATE
```

Comment:
What's the difference between these two tests? maybe the second one should be `enrollment_start` instead of just `start`?